MagaluCloud/magalu
# feat(tf): add more property setters to DBaaS and VM resources

## Description

This has some general fixes in TF and it adds property setters for some DBaaS and VM resources. You can check the documentation diff to see what changed

## Related Issues

- Closes #655

## How to test it

- DBaaS instance
  - Create an instance with a certain `volume` and `flavor_id`
  - Change the `volume` or the `flavor_id` or both
  - Apply and see if successful
- DBaaS replicas
  - Create a replica with a certain `status` (or omit the `status`)
  - Change the `status` to either `ACTIVE` or `STOPPED`
  - Apply and see if successful
- VM Snapshot
  - Create a snapshot with a certain `name`
  - Change the `name`
  - Apply and see if successful
